docs/source/07_extend_kedro/03_custom_datasets.md

```diff
@@ -36,11 +36,11 @@ Here is an example skeleton for `ImageDataSet`:
 <summary><b>Click to expand</b></summary>
 
 ```python
-from typing import Any, Dict, List
+from typing import Any, Dict
 
 import numpy as np
 
-from kedro.io import AbstractDataSet
+from kedro.io.core import AbstractDataSet
```

Comment:
```suggestion
from kedro.io import AbstractDataSet
```

This is a good catch. We should actually prefer the top-level imports so that if we rename or move `core`, this import would still work.. but if it's a faff to recreate the screenshot then I'm not that bothered.

Reply:
This is a good point, I'll change it. It's no faff to recreate the screenshot any more since I made it a code `diff` rather than a picture 😀

Reply:
Done in 7b1a46b